docs/files/content/style.css

```diff
@@ -5,71 +5,71 @@
 /*--------------------------------------------------------------------------*/
 
 /* strings --- and stlyes for other string related formats */
-span.s { color:#E0E268; }
+span.s { color:#60815f; }
```

Comment:
We probably should leave that to someone else (I mean making F# code look nice). Ideally I'd like to steal/import code from ionide for example.